### PR TITLE
Add CondIsVectorNormalized

### PR DESCRIPTION
**Pull request by youngmasterseth**
### Description

Add is vector normalized property condition
---
**Target Minecraft Versions:** Any
**Requirements:** 1.13+
**Related Issues:** n/a

### CODE REVIEW

File: src/main/java/ch/njol/skript/conditions/CondIsVectorNormalized.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+@Name("Is Normalized")
+@Description("Checks whether a vector is normalized i.e. length of 1")
+@Examples("vector of player's location is normalized")
+@Since("INSERT VERSION")
+@RequiredPlugins("1.13 or newer")
+public class CondIsVectorNormalized extends PropertyCondition<Vector> {
+	
+	static {
+		register(CondIsVectorNormalized.class, "normalized", "vectors");
```
Comment (APickledWalrus) [inline src/main/java/ch/njol/skript/conditions/CondIsVectorNormalized.java:38]: If this only works on 1.13+ you should be checking whether the method exists.
e.g.
```java
Skript.methodExists(Vector.class, "isNormalized")
```

**Review by APickledWalrus — CHANGES_REQUESTED** [review]

File: src/main/java/ch/njol/skript/conditions/CondIsVectorNormalized.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	import ch.njol.skript.doc.Examples;
+	import ch.njol.skript.doc.Name;
+	import ch.njol.skript.doc.RequiredPlugins;
+	import ch.njol.skript.doc.Since;
+
+@Name("Is Normalized")
+@Description("Checks whether a vector is normalized i.e. length of 1")
+@Examples("vector of player's location is normalized")
+@Since("INSERT VERSION")
+@RequiredPlugins("1.13 or newer")
```
Comment (APickledWalrus) [inline src/main/java/ch/njol/skript/conditions/CondIsVectorNormalized.java:34]: Looks like this was added in 1.13.2 from what I can see on the javadocs
```suggestion
@RequiredPlugins("1.13.2 or newer")
```

File: src/main/java/ch/njol/skript/conditions/CondIsVectorNormalized.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+@Name("Is Normalized")
+@Description("Checks whether a vector is normalized i.e. length of 1")
+@Examples("vector of player's location is normalized")
+@Since("INSERT VERSION")
+@RequiredPlugins("1.13.2 or newer")
+public class CondIsVectorNormalized extends PropertyCondition<Vector> {
+	
+	static {
+		if (Skript.methodExists(Vector.class, "isNormalized")) {register(CondIsVectorNormalized.class, "normalized", "vectors");}
```
Comment (APickledWalrus) [inline src/main/java/ch/njol/skript/conditions/CondIsVectorNormalized.java:40]: Formatting needs fixed
```suggestion
		if (Skript.methodExists(Vector.class, "isNormalized")) {
			register(CondIsVectorNormalized.class, "normalized", "vectors");
		}
```

**Review by APickledWalrus — CHANGES_REQUESTED** [review]

**Review by DeceptionYMS — APPROVED** [review]

File: src/main/java/ch/njol/skript/conditions/CondIsVectorNormalized.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
```
Comment (ShaneBeee) [inline src/main/java/ch/njol/skript/conditions/CondIsVectorNormalized.java:18]: bensku has updated the last line of the license header, you can find the update version [**HERE**](https://github.com/SkriptLang/Skript/blob/master/licenseheader.txt) and update accordingly

File: src/main/java/ch/njol/skript/conditions/CondIsVectorNormalized.java [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+	import ch.njol.skript.doc.Examples;
+	import ch.njol.skript.doc.Name;
+	import ch.njol.skript.doc.RequiredPlugins;
+	import ch.njol.skript.doc.Since;
+
+@Name("Is Normalized")
+@Description("Checks whether a vector is normalized i.e. length of 1")
+@Examples("vector of player's location is normalized")
+@Since("INSERT VERSION")
+@RequiredPlugins("1.13.2 or newer")
```
Comment (ShaneBeee) [inline src/main/java/ch/njol/skript/conditions/CondIsVectorNormalized.java:36]: To keep consistency with other classes, we should probably be doing "Minecraft 1.13.2+" here

**Review by ShaneBeee — CHANGES_REQUESTED** [review]
Just a couple small changes

**Pull request closed without merging**